src/QsCompiler/QirGeneration/Subtransformations/ExpressionKindTransformation.cs

```diff
@@ -584,16 +658,21 @@ Value ArrayToString(ArrayValue array)
                 var ty = evaluated.QSharpType.Resolution;
                 if (ty.IsString)
                 {
-                    UpdateStringRefCount(evaluated.Value, 1);
-                    return evaluated.Value;
+                    Value quote = CreateConstantString("\"");
```

Comment:
Nit: Is there a reason why `quote` was created using the `Value` type instead of `var`? I see that the `comma` variable a few lines above uses the same pattern.

Reply:
I think this might have been left from an earlier stage of the code where the explicit Value was needed for the nullcheck to be happy. I'll change it to var.